galleries/examples/mplot3d/intersecting_planes.py

```diff
@@ -0,0 +1,89 @@
+"""
+===================
+Intersecting planes
+===================
+
+This examples demonstrates drawing intersecting planes in 3D. It is a generalization
+of :doc:`/gallery/mplot3d/imshow3d.py`.
+
+Drawing intersecting planes in `mplot3d` is complicated, because `mplot3d` is not a
```

Comment:
```suggestion
Drawing intersecting planes in `.mplot3d` is complicated, because `.mplot3d` is not a
```

maybe that will fix the cross referencing?